### DISCUSSION TITLE
Add pt-br language

### DISCUSSION THREAD

**Discussion by cyberrep** [openwrt-xiaomi/xmir-patcher#11] [General]
I have the lmo (ax3000) and upload to the directory and its works

If you wanna add

[base.pt.zip](https://github.com/openwrt-xiaomi/xmir-patcher/files/13218042/base.pt.zip)

I just not found how to convert from lmo to po
in this case I just add at install_lang.py
gw.upload(fn_dir + 'base.pt.lmo', '/tmp/' + 'base.pt.lmo') # force pt-br

and add at lang_patch.sh
uci set luci.languages.pt=Portugues #add pt-br

**Comment by lanke521** [comment]
I would like to ask Xiaomi 5G CPE Pro (AX5400), can this router enable ssh?
